_posts/2020-02-26-ion-java-1_6_0-released.md

```diff
@@ -9,6 +9,5 @@ This release includes:
 * Optimizations for reading and writing binary strings.
 * A fix for a bug that could cause an infinite loop when an IonValue was modified while simultaneously being read with a TreeReader.
 
-| [Release Notes](https://github.com/amzn/ion-java/releases/tag/v1.6.0)
-| [Ion Java](https://github.com/amzn/ion-java) |
+| [Release Notes](https://github.com/amzn/ion-java/releases/tag/v1.6.0) | [Ion Java](https://github.com/amzn/ion-java) |

```

Comment:
Just an observation: this doesn't change the way the file is rendered, but it does make this line of source long enough that it has to wrap. Maybe a good compromise is to put the trailing `|` on its own line instead?

Reply:
Turns out this does change the rendered view.  Note how the "buttons" are oriented [here](http://amzn.github.io/ion-docs/news/ion-java/2020/02/26/ion-java-1_6_0-released.html) vs. [here](http://amzn.github.io/ion-docs/news/2019/05/23/ion-hash-released.html).  Leaving as-is.